dependencies.yml

```diff
@@ -4,8 +4,8 @@
 #
 boms:
 - com.fasterxml.jackson:jackson-bom:2.11.2
-- com.linecorp.armeria:armeria-bom:0.99.9
-- io.micrometer:micrometer-bom:1.5.3
+- com.linecorp.armeria:armeria-bom:1.0.0
```

Comment:
Update Javadoc link to `https://www.javadoc.io/doc/com.linecorp.armeria/armeria-javadoc/1.0.0/`?

Reply:
Oops Thanks!